src/rollup-plugin-deno-resolver/denoResolver.ts

```diff
@@ -50,12 +49,16 @@ export function denoResolver(
       const code = await loadUrl(url, fetchOpts);
```

Comment:
Given we are no longer using the `code` when dealing with TypeScript, this should be moved after the `if` to save us a redundant `fetch` / `readFile`.